src/reporting/parameter.ts

```diff
@@ -99,15 +90,14 @@ export function createStackParametersQuickSight(scope: Construct, paramGroups?:
   };
 
   const quickSightTimezoneParam = new CfnParameter(scope, 'QuickSightTimezoneParam', {
-    description: 'Timezone of the builtin dashboard.',
+    description: 'The time zone with app id',
     type: 'String',
-    default: 'UTC',
+    default: '[]',
```

Comment:
@tyyzqmf Is it same format with data modeling?

Reply:
confirmed with @tyyzqmf , It is same format with modeling.